orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -379,9 +413,7 @@ def _convert_backward_input_list_to_dict(self, *inputs):
 
         return result
 
-
-    @staticmethod
-    def _get_forward_graph(module, *inputs, **kwargs):
+    def _get_forward_graph(self, module, input_names, dynamic_axes, *inputs, **kwargs):
```

Comment:
What is the rationale in making it not static and still pass `module` as argument? This parameter could be suppressed and internally `self._original_module`, right?

Reply:
you're right. I initially wanted to keep it static and then realized it needn't be static. Missed removing the module argument. Will remove that.